src/middlewared/middlewared/plugins/failover_/remote.py

```diff
@@ -161,7 +161,7 @@ def _sub_callback(self, name, type_, **message):
             except Exception:
                 logger.warning('Failed to run callback for %s', name, exc_info=True)
 
-    def send_file(self, token, local_path, remote_path):
+    def send_file(self, token, local_path, remote_path, options={}):
```

Comment:
It's not common to specify `options={}` in the function's kwarg section. Let's set it to `options=None` and then inside the function do something like
```
options = options or {}